AzureMaps/AzureMaps.php

```diff
@@ -0,0 +1,312 @@
+<?php
+/**
+ * Created by PhpStorm.
+ * User: Max Langerman
+ * Date: 7/13/20
+ * Time: 12:11 AM
+ */
+
```

Comment:
Can we use strict types please
```suggestion
declare(strict_types=1);
```

(and on other files)